src/vmm/src/lib.rs

```diff
@@ -299,10 +309,13 @@ impl TryFrom<VMMConfig> for Vmm {
         #[cfg(target_arch = "aarch64")]
         let fdt_builder = FdtBuilder::new();
 
+        let irq_allocator = IrqAllocator::new(4, vm.max_irq())?;
```

Comment:
We should have a define for `4` instead and use it when we initialize the console on both platforms.